INSTRUCTIONS.md

```diff
@@ -2,66 +2,83 @@
 
 Other packages we will use in this workshop are:
 
-* [scikit-learn](http://scikit-learn.org/stable/): a machine learning library
-* [pyZMQ](https://zeromq.github.io/pyzmq/): the Python binding for ZMQ, a simple communication library
+* [scikit-learn](http://scikit-learn.org/stable/): a machine learning library.
+* [pyZMQ](https://zeromq.github.io/pyzmq/): the Python binding for ZMQ, a simple communication library.
 
-To install Python 2 or Python 3, and the required packages, we suggest you download and install the [Anaconda distribution](http://continuum.io/downloads). Anaconda is a Python distribution that includes Python 2.7 (in case of Anaconda2) or Python 3.5 (in case of Anaconda 3), all the packages we will need for the workshop (as well as plenty other useful packages), and [Spyder](https://pythonhosted.org/spyder/), a great IDE for scientific computing in Python.
+To install Python 2 or Python 3, and the required packages, we suggest you download and install the [Anaconda distribution](http://continuum.io/downloads). Anaconda is a Python distribution that includes Python 2.7 (in case of Anaconda 2) or Python 3.5 (in case of Anaconda 3), all the packages we will need for the workshop (as well as plenty other useful packages), and [Spyder](https://pythonhosted.org/spyder/), a great IDE for scientific computing in Python.
 
-#### 1.1 Installation with Anaconda (recommended)
+#### (1) Installation of Python with Anaconda (recommended)
 
 1. Download the [Anaconda graphical installer](http://continuum.io/downloads) (if your Windows version is 32-bit, make sure to download the 32-bit installer).
 2. Execute the installer.
 
 This installs Python, Spyder and all the packages we will need for the workshop (including scikit-learn and pyZMQ).
 
-#### 1.2 Individual installation of Python and packages
-Alternatively, you can [download Python 2.7 independently](http://docs.python-guide.org/en/latest/starting/install/win/). Make sure to install ```pip``` (as explained [here]((http://docs.python-guide.org/en/latest/starting/install/win/))) and grab NumPy, matplotlib and scikit-learn by calling ```pip install <package_name>``` on the command line (or any other way you prefer). Make sure you have a text editor or IDE you can work with as well.
-
-### 2. Muse SDK
+#### (2) Individual installation of Python and packages
+Alternatively, you can [download Python 2.7 independently](http://docs.python-guide.org/en/latest/starting/install/win/). Make sure to install ```pip``` (as explained [here]((http://docs.python-guide.org/en/latest/starting/install/win/))) and grab **NumPy**, **matplotlib** and **scikit-learn** by calling ```pip install <package_name>``` on the command line (or any other way you prefer). Make sure you have a text editor or IDE you can work with as well.
```

Comment:
Another way that package names can be emphasized in markdown is with the `...` (weird single quotes?) character (just in case you haven't used it before).